docs/content/en/docs/reference/artifacts.md

```diff
@@ -130,7 +132,7 @@ The following procedure describes how to use `image-builder` to build images for
 
 To use `image-builder` you must meet the following prerequisites:
 
-* Run on Ubuntu 22.04 or later 
+* Run on Ubuntu 22.04 or later (for Ubuntu images) or RHEL 8.4 or later (for RHEL images)
```

Comment:
We will only run on Ubuntu 22.04 for any os building. Lets remove this change.

Reply:
Sorry. That was an early misconception I forgot to fix. Fixed now.